packages/plugin-webpack/plugin.js

```diff
@@ -223,19 +219,23 @@ module.exports = function plugin(config, args) {
 
         //Now that webpack is done, modify the html file to point to the newly compiled resources
         Object.keys(entries).forEach((name) => {
-          if (entrypoints[name] !== undefined) {
-            const assetFiles = chain(entrypoints, [name, "assets"]);
-            const script = entries[name].script;
-            const jsFile = assetFiles.find((d) => d.endsWith(".js"));
-            const cssFile = assetFiles.find((d) => d.endsWith(".css"));
-            script.removeAttribute("type");
-            script.src = path.posix.join(baseUrl, jsFile);
-            if (cssFile) {
-              let csslink = dom.window.document.createElement("link");
-              csslink.setAttribute("rel", "stylesheet");
-              csslink.href = path.posix.join(baseUrl, cssFile);
-              dom.window.document.querySelector("head").append(csslink);
+          const originalScriptEl = entries[name].script;
```

Comment:
Cleaned this whole chunk up and added support for multiple built assets, which is common with core-js (where the first entrypoint loads both its own `index.[hash]` chunk, and some polyfills from the vendor chunk).